defaults.go

```diff
@@ -272,18 +272,24 @@ func IsMsg(buf []byte) error {
 
 // IsFqdn checks if a domain name is fully qualified.
 func IsFqdn(s string) bool {
-	s2 := strings.TrimSuffix(s, ".")
-	if s == s2 {
+	// Check for (and remove) a trailing dot, returning if there isn't one.
```

Comment:
```suggestion
	// Check for (and remove) a trailing dot, rejecting if there isn't one.
```

Reply:
I think return is clearer, we're not validating or otherwise approving the input, only answering a question regarding it.